modules/indexExchangeBidAdapter.js

```diff
@@ -663,11 +815,305 @@ var IndexExchangeAdapter = function IndexExchangeAdapter() {
+      bid.params.video.minduration = VIDEO_OPTIONAL_PARAMS_MAP.minduration;
+    }
+
+    let startdelay = bid.params.video.startdelay;
+    if (typeof startdelay === 'undefined' || !isValidStartDelay(startdelay)) {
+      utils.logInfo(`Using default value for 'startdelay', default: ${VIDEO_OPTIONAL_PARAMS_MAP.startdelay}`);
+      bid.params.video.startdelay = VIDEO_OPTIONAL_PARAMS_MAP.startdelay;
+    }
+
+    let linearity = bid.params.video.linearity;
+    if (typeof linearity === 'undefined' || !isValidLinearity(linearity)) {
+      utils.logInfo(`Using default value for 'linearity', default: ${VIDEO_OPTIONAL_PARAMS_MAP.linearity}`);
+      bid.params.video.linearity = VIDEO_OPTIONAL_PARAMS_MAP.linearity;
+    }
+
+    let mimes = bid.params.video.mimes;
+    let playerType = bid.params.video.playerType.toUpperCase();
+    if (typeof mimes === 'undefined' || !isValidMimeArray(mimes)) {
+      utils.logInfo(`Using default value for 'mimes', player type: '${playerType}', default: ${DEFAULT_MIMES_MAP[playerType]}`);
+      bid.params.video.mimes = DEFAULT_MIMES_MAP[playerType];
+    }
+
+    let apiList = bid.params.video.apiList;
+    if (typeof apiList !== 'undefined' && !isValidApiArray(apiList, playerType)) {
+      utils.logInfo(`Removing invalid api versions from api list.`);
+      if (utils.isArray(apiList)) {
+        bid.params.video.apiList = apiList.filter(api => SUPPORTED_API_MAP[playerType].includes(api));
+      } else {
+        bid.params.video.apiList = [];
+      }
+    }
+
+    if (typeof apiList === 'undefined' && bid.params.video.allowVPAID && utils.isA(bid.params.video.allowVPAID, 'Boolean')) {
+      bid.params.video.mimes = bid.params.video.mimes.concat(DEFAULT_VPAID_MIMES_MAP[playerType]);
+      bid.params.video.apiList = SUPPORTED_API_MAP[playerType];
+    }
+
+    if (utils.isEmpty(bid.params.video.apiList)) {
+      utils.logInfo(`API list is empty, VPAID ads will not be requested.`);
+      delete bid.params.video.apiList;
+    }
+
+    delete bid.params.video.playerType;
+    delete bid.params.video.allowVPAID;
+
+    return bid;
+  }
+
+  /* Turn bid request sizes into ut-compatible format */
+  function getSizes(requestSizes) {
+    let sizes = [];
+    let sizeObj = {};
+
+    if (utils.isArray(requestSizes) && requestSizes.length === 2 && !utils.isArray(requestSizes[0])) {
+      if (!utils.isNumber(requestSizes[0]) || !utils.isNumber(requestSizes[1])) {
+        return sizes;
+      }
+      sizeObj.width = requestSizes[0];
+      sizeObj.height = requestSizes[1];
+      sizes.push(sizeObj);
+    } else if (typeof requestSizes === 'object') {
+      for (let i = 0; i < requestSizes.length; i++) {
+        let size = requestSizes[i];
+        sizeObj = {};
+        sizeObj.width = parseInt(size[0], 10);
+        sizeObj.height = parseInt(size[1], 10);
+        sizes.push(sizeObj);
+      }
+    }
+
+    return sizes;
+  }
+
   return {
     callBids: _callBids
   };
```

Comment:
You need to return the original `this` instance.

You can use
```
return Object.assign(this, {
  callBids: _callBids
});
```
to accomplish this